.gitignore

```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules
+
+*.md
```

Comment:
- [ ] You have done very well so far however in the requirements you were tasked to add the test.md file to the .gitignore file. The goal of this is to make sure that this file is not tracked by git (ie. git doesn't pay attention to this file). If done properly, the test.md file should not show on github:

![image](https://user-images.githubusercontent.com/107795045/218882161-4eadfce8-ad56-4449-832b-c5bc13f5817b.png)

To do this, replace ```*.md``` with ```test.md```